### ISSUE TITLE
Outstanding PRs

### ISSUE THREAD

**Issue by matthew-dean** [less/less-meta#11]
Can we focus on reviewing our outstanding PRs? I have one open that dramatically improves loading time in the browser, but I wondered if there might be any downside to temporarily hiding the page while rendering rather than sync ajax. I don't think there is, but it's one thing that just needs review.

There's not a long list of open PRs (14), but there are some that have been open since 2013.

**Comment by seven-phases-max** [comment]
>I have one open that dramatically improves loading time in the browser, 

I guess we should just merge it. The bottleneck is that probably none of us have any projects (or something like that) involving the browser-side script, thus we can't be really helpful, thus the only way to actually test is to release it and wait for issue...

**Comment by matthew-dean** [comment]
Okay. It *seems* low risk, unless there's a one-off browser that is weird about updating styling. And yes, I've been using this fix in a project for a while. I will merge it.

**Comment by matthew-dean** [comment]
That's merged. It still would be good to go through the other PRs.

**Comment by seven-phases-max** [comment]
As for other PRs I'm afraid each one need a separate decision. For instance two `contrast` PR are stalled because we did not decide on "applying breaking-changes" strategy for it yet. Same for others (`extend` improvement PR is stalled because it needs additional work, `property as variables` needs benchmarking if I recall correctly etc.)

So maybe I would just suggest to go through the list and ping those one finds to be "no harm to merge" and then, if no objections come soon, finally merge it. And for others we need to add a comment and/or add a label why it's not merged yet ("postpone for v3", "needs additional work", "reject because of out-dated code-base" etc.).

Well, you know, all that sort of dirty and boring bureaucratic work @lukeapage usually did before :)

**Comment by matthew-dean** [comment]
Is @lukeapage really busy these days? Or is he out?

**Comment by matthew-dean** [comment]
This does't really need to still be open, so closing.
